openstack/resource_openstack_compute_instance_v2.go

```
@@ -475,6 +488,32 @@ func resourceComputeInstanceV2Create(d *schema.ResourceData, meta interface{}) e
 			server.ID, err)
 	}
 
+	if d.Get("state").(string) != "" {
+		vmState := d.Get("state").(string)
+		if strings.ToLower(vmState) == "shutoff" {
+			err = startstop.Stop(computeClient, d.Id()).ExtractErr()
+			if err != nil {
+				log.Printf("[WARN] Error stopping OpenStack instance: %s", err)
```

If the action failed, we should return an error rather than log one.